src/moments.jl

```diff
@@ -363,3 +363,44 @@ end
 
 kurtosis(v::RealArray) = kurtosis(v, mean(v))
 kurtosis(v::RealArray, wv::AbstractWeights) = kurtosis(v, wv, mean(v, wv))
+
+"""
+    cumulant(v, k, [wv::AbstractWeights], m=mean(v))
+
+Return the `k`th order cumulant of a real-valued array `v`, optionally
+specifying a weighting vector `wv` and a center `m`. This quantity is calculated using a recursive definition on lower order cumulants and central moments.
+"""
+function cumulant(v::RealArray, k::Int, m::Real)
```

Comment:
```suggestion
function cumulant(v::RealArray, k::Int, m::Real=mean(v))
```

Reply:
Maybe `m` could even be a keyword argument (probably with a more descriptive name such as `mean`)?

Reply:
Yeah that would be better, but `moment` currently has a positional argument (remnants from pre-Julia 1.0 times), so we should change both at the same time.